lib/ansible/cli/__init__.py

```diff
@@ -798,3 +798,20 @@ def _play_prereqs(options):
         variable_manager.options_vars = load_options_vars(options, CLI.version_info(gitinfo=False))
 
         return loader, inventory, variable_manager
+
+    def _load_callback(self):
+
+        from ansible.plugins.loader import callback_loader
```

Comment:
Are we importing this here due to instantiation order?  If so, might want to document that.

Reply:
was quick hack to see if it worked, was thinking on adding a toggle to avoid the extra import, but not sure on how it should end up looking